plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/MavenFacetImpl.java

```diff
@@ -234,7 +234,6 @@ private void putArtifact(final MavenPath path, final Payload payload, final Stor
           .group(coordinates.getGroupId())
           .name(coordinates.getArtifactId())
           .version(coordinates.getVersion());
-      component.formatAttributes().set(StorageFacet.P_PATH, path.getPath());
```

Comment:
@kellyrob99 Yes, component:asset is 1:n, and component represents GAV while asset (implicitly) GAV + CE and holds complete path too.